### PR TITLE
Adds a test about flow.pkts_toclient keyword

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6164

Modifies #1261 with more keywords tested

### CODE REVIEW

File: tests/detect-flow-pkts/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Test
+====
+
+Test `flow.pkts_toclient` keyword
```
Comment (jufajardini) [inline tests/detect-flow-pkts/README.md:4]: I think it's good to list the other words that are being tested :P

**Review by jufajardini — CHANGES_REQUESTED** [review]
Just one comment, so we have a better test description on the readme :)

File: tests/detect-flow-pkts/README.md (hunk 1 above)
Comment (catenacyber) [reply tests/detect-flow-pkts/README.md]: Just force-pushed a new version, is it ok ?

Comment (jufajardini) [reply tests/detect-flow-pkts/README.md]: Works for me :P

**Review by jufajardini — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #1468, thanks!

**Pull request closed without merging**